src/shared/messages/ErrorMessageEnum.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+
+class ErrorMessageEnum(Enum):
```

Comment:
this is a minor, but maybe we should consider having an `enum.py` file inside this `shared` folder.
In python we usually name files in `snack_case`

Reply:
I will keep this for a while because I believe that we will have more shared enums in the application and I would like to keep at least the ones for messages separated. But i will rename the file to `snack_case`